### ISSUE TITLE
Request: Make it read the descriptions of posts

### ISSUE THREAD

**Issue by Alex-Sour** [charlypoirier/redditube#5]
I would love to be able to have posts from subs like r/EntitledParents and r/TalesFromTechSupport be readable with this!
Also sorry for all the issues lmao

**Comment by charlypoirier** [comment]
Yeah that's a good idea, I don't have a lot of free time to work on it but I'll keep that in mind.
No problem with the issues, it makes the repository more active lol.
